src/components/App/App.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import BreadcrumbsContextProvider from '../../contexts/BreadcrumbsContext';
```

Comment:
To get rid of these hard-to-read and write paths ( ../../someComponent ) you can use aliases (@someComponent). They can be configured in the webpack.config and tsconfig.json for typing to work correctly